code/modules/crafting/quality_of_crafting/crafting.dm

```diff
@@ -135,6 +135,20 @@
 	craftdiff = 0
 	uses_attacked_atom = FALSE
 
+/datum/repeatable_crafting_recipe/crafting/stake
```

Comment:
```suggestion
/datum/repeatable_crafting_recipe/crafting/caningstick
```
You have the same recipe typepath as the wooden stake, which makes stakes not craftable.